### PR TITLE
NET5.0 support

### PR DESCRIPTION
**Pull request by MaceWindu**

### CODE REVIEW

File: NuGet/linq2db.EntityFrameworkCore.5.0.nuspec [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		<copyright>Copyright © 2020 Igor Tkachev, Ilya Chudin, Svyatoslav Danyliv, Dmitry Lukashenko</copyright>
+		<description>Allows to execute Linq to DB (linq2db) queries in Entity Framework Core DbContext.</description>
+		<summary />
+		<tags>linq linq2db LinqToDB ORM database entity-framework-core EntityFrameworkCore EFCore DB SQL SqlServer SqlCe SqlServerCe MySql Firebird SQLite Oracle ODP PostgreSQL DB2</tags>
+		<requireLicenseAcceptance>false</requireLicenseAcceptance>
+		<icon>images\icon.png</icon>
+		<projectUrl>https://github.com/linq2db/linq2db.EntityFrameworkCore</projectUrl>
+		<license type="file">MIT-LICENSE.txt</license>
+		<dependencies>
+			<group targetFramework=".NETStandard2.0">
```
Comment (MaceWindu) [inline NuGet/linq2db.EntityFrameworkCore.5.0.nuspec:17]: should be 2.1

**Comment by Shane32** [comment]
Is the plan to remove support for EF Core 3.x entirely?

**Comment by sdanyliv** [comment]
@Shane32, no, just separate by branches. It’s too early to stop supporting.

**Comment by Shane32** [comment]
Ok that makes sense.  I just noticed this PR pulls into `master` and was a bit surprised.  Perhaps a note in the PR that says "do not merge" or something would be in order.

**Pull request merged**